cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -525,8 +526,8 @@ private String rebalanceStateConditionStatus(KafkaRebalanceStatus status) {
                                     p.complete(new KafkaRebalanceStatusBuilder()
                                             .withSessionId(null)
                                             .addNewCondition()
-                                                .withNewStatus(State.Stopped.toString())
-                                                .withNewType(KafkaRebalanceStatus.REBALANCE_STATUS_CONDITION_TYPE)
+                                                .withNewType(State.Stopped.toString())
+                                                .withNewStatus(KafkaRebalanceStatus.REBALANCE_STATUS_CONDITION_STATUS)
```

Comment:
I was noticing that adding a condition in this way isn't not calling the `.withLastTransitionTime(iso8601Now())` which is called by all the `StatusUtils` method. 
I was wondering if we should have a `buildCondition` as we have the `buildWarningCondition` passing just the `type` and it will set the status to "True" instead of passing again this constant.